pkg/meta/dump.go

```diff
@@ -264,6 +272,16 @@ func (dm *DumpedMeta) writeJsonWithOutTree(w io.Writer) (*bufio.Writer, error) {
 	return bw, nil
 }
 
+func (m *baseMeta) loadDumpedQuotas(ctx Context, quotas map[Ino]*DumpedQuota) {
```

Comment:
It may be difficult, as we parse and load dumped JSON file streamingly, we cannot ensure the quotas are loaded before parsing entries.

I prefer to re-calculate quotes during dumping.

Reply:
Ok, now we will calculate quote during loading